### DISCUSSION TITLE
Addition of stiffening member throws an error

### DISCUSSION THREAD

**Discussion by Idonotuse** [idea-statica/ideastatica-public#753] [Help Q&A]
I'm trying create an IOM which has multiple members along with one stiffening member. When the stiffening member is added as a regular member there aren't any issues.

![Screenshot 2024-12-20 111903](https://github.com/user-attachments/assets/70225646-fca2-4512-b216-034d08ca5ca0)

But whenever I try to add the member as a stiffening member by doing the following

![image](https://github.com/user-attachments/assets/f0d78599-688b-442c-b6d8-b2db7fec1657)

then check bot throws following error

![image](https://github.com/user-attachments/assets/46832a54-8369-44d0-a7fb-b4aaf1bbb1b5)

I've checked in both python as well as C# but the issue remains the same. I've attached the xml file for your reference.
[idea_result.zip](https://github.com/user-attachments/files/18207383/idea_result.zip)

Can you help me out?

**Comment by nathanlukeIDEA** [comment]
@mpospisil @OndraZitko
